### DISCUSSION TITLE
Qcodes Instrument View Help

### DISCUSSION THREAD

**Discussion by Lixuanqiu** [microsoft/Qcodes#6690] [Q&A]
Hi, I am a beginner in qcodes and I am writing a qcodes control program for our lab. We've been using qtlab for a while (I'm told this is the precursor to qcodes?). I would like to know whether qcodes has a window similar to "Instrument View" in qtlab, which can check the status of Instruments in real time?
![image](https://github.com/user-attachments/assets/db19d4e3-d1e6-4bfd-bae1-800514667ddf)

**Comment by jenshnielsen** [comment]
Hi, 

QCoDeS ships with a webbased monitor that enables you to list a number of parameters that you wish to monitor and then open a webpage to list them https://microsoft.github.io/Qcodes/api/monitor/index.html#module-qcodes.monitor

**Reply by Lixuanqiu** [reply]
Thanks！
